app/services/ccms/base_requestor.rb

```diff
@@ -30,13 +30,15 @@ def transaction_request_id
     # :nocov:
     def soap_client
       @soap_client ||= Savon.client(
+        headers: { 'x-api-key' => Rails.configuration.x.ccms_soa.aws_gateway_api_key },
```

Comment:
❓ is going to work on `Sit10` after merging this? (I guess will just ignore the empty header). Can anyone who currently can run the testing script correctly over `Sit10` check if it works on this branch?

Reply:
Mark is testing this.